app/views/owned_tag_sets/show_options.html.erb

```diff
@@ -1,18 +1,28 @@
-<% cache("tagsets_#{@tag_set_ids.join('-')}_#{@tag_sets.collect(&:updated_at).sort.last}", skip_digest: true) do %>
-  <h2>
-    <%= ts("%{tag_type} Options", tag_type: tag_type_label_name(tag_type)) %>
-    <%= ts("From") %>:</h2>
-    <ul class="commas">
-      <% @tag_sets.each do |ts| %>
-        <li><%= link_to ts.title, tag_set_path(ts) %></li>
-      <% end %>
-    </ul>
+<% # ids of tag sets whose tags are being shown %>
+<% ids = @tag_set_ids.join("-") %>
+
+<% # the show tag sets' most recent updated_at %>
+<% newest_updated_at = @tag_sets.collect(&:updated_at).sort.last %>
+
+<% # cache key for all of the sets' tags, which expires when the number of tags changes or when the most recent updated_at changes %>
+<% tags_cache_key = Tag.tag_sets_cache_key(@tag_type, @tag_set_ids) %>
```

Comment:
Would it work to call `@tags.cache_key` here? I think it's defined pretty similarly to `tag_sets_cache_key`, with the exception of the call to `by_name_without_articles`:
https://github.com/otwcode/otwarchive/blob/67b7b149d41ef626aca3596f2b1448b34d0cdc05/app/controllers/owned_tag_sets_controller.rb#L64

Reply:
Ohh, yeah, much better than adding a new method -- good catch!